tutorials/autotvm/tune_relay_x86.py

```diff
@@ -114,6 +114,17 @@ def get_network(name, batch_size):
 # We will use local mode for tuning configuration. RPC tracker
 # mode can be setup similarly to the approach in
 # :ref:`tune_relay_arm` tutorial.
+#
+# In the measure option, we turn on enable_cpu_cache_flush to
+# get more precise measurement. When we turn it on, we don't
+# need to set min_repeat_ms to dynamically adjust to run op
+# many times so that we get precise measurement as when we
+# have cache flush, we could get precise measurement even we
+# run serveral times. So, we could just set number be 1 and
+# repeat be 10 to run only 10 times. The reason we set number be 1
+# is we will turn on cache flush before every repeat run in
+# internal implementation. So if number is greater than 1, the
+# cache flush effect will be probably invalid.
```

Comment:
This paragraph explains the motivation of this PR. But it is very hard for new users to understand.
Because this tutorial is for new users with few tvm experiences, I suggest the follwoing modification.

Reply:
```suggestion
# To perform a precise measurement, we should repeat the measurement several times and
# use the average of results.  In addition, we need to flush the cache for the weight tensors
# between repeated measurements. This can make the measured latency of one operator 
# closer to its actual latency during end-to-end inference.
```